5-analysis/analysis-powder-diffraction.ipynb

```diff
@@ -0,0 +1,1665 @@
+   ],
+   "outputs": [],
+   "execution_count": null
+  },
+  {
+   "cell_type": "markdown",
+   "id": "60e8f0fc",
+   "metadata": {},
+   "source": [
+    "### 🚀 Exercise 5: Analyze and Fit the Data\n",
+    "\n",
+    "#### Exercise 5.1: Set Fit Parameters\n",
+    "\n",
+    "Select the parameters to be refined during the fitting process.\n",
+    "\n",
+    "**Hint:** You can start with the same parameters as in the Si fit, but\n",
+    "this time you will refine the scale factor of the LBCO phase and the\n",
+    "background points for the second simulation.\n",
+    "\n",
+    "**Solution:**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "id": "72ba7c10",
+   "metadata": {},
+   "source": [
+    "project_2.experiments[\"sim_lbco\"].linked_phases[\"lbco\"].scale.free = True\n",
+    "for line_segment in project_2.experiments[\"sim_lbco\"].background:\n",
+    "    line_segment.y.free = True"
+   ],
+   "outputs": [],
+   "execution_count": null
+  },
+  {
+   "cell_type": "markdown",
+   "id": "76151a8e",
+   "metadata": {},
+   "source": [
+    "#### Exercise 5.2: Run Fitting\n",
+    "\n",
+    "Visualize the measured and calculated diffraction patterns before fitting and\n",
+    "then run the fitting process.\n",
+    "\n",
+    "**Hint:** Use the `plot_meas_vs_calc` method of the project to visualize the\n",
+    "measured and calculated diffraction patterns before fitting. Then, use the\n",
+    "`fit` method of the `analysis` object of the project to perform the fitting\n",
+    "process.\n",
+    "\n",
+    "**Solution:**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "id": "17d6fab3",
+   "metadata": {},
+   "source": [
+    "project_2.plot_meas_vs_calc(expt_name=\"sim_lbco\")"
+   ],
+   "outputs": [],
+   "execution_count": null
+  },
+  {
+   "cell_type": "code",
+   "id": "cab87e11",
+   "metadata": {},
+   "source": [
+    "project_2.analysis.fit()"
+   ],
+   "outputs": [],
+   "execution_count": null
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3e8311ca",
+   "metadata": {},
+   "source": [
+    "#### Exercise 5.3: Find the Misfit in the Fit\n",
+    "\n",
+    "Visualize the measured and calculated diffraction patterns after the fit. As\n",
+    "you can see, the fit shows noticeable discrepancies. If you zoom in on different regions of the pattern,\n",
+    "you will observe that all the calculated peaks are shifted to the left.\n",
+    "\n",
+    "What could be the reason for the misfit?\n",
+    "\n",
+    "**Hint**: Consider the following options:\n",
+    "1. The conversion parameters from TOF to d-spacing are not correct.\n",
+    "2. The lattice parameters of the LBCO phase are not correct.\n",
+    "3. The peak profile parameters are not correct.\n",
+    "4. The background points are not correct.\n",
+    "\n",
+    "**Solution**:\n",
+    "1. ❌ The conversion parameters from TOF to d-spacing were set based on the data reduction process and this was verified in the Si fit.\n",
```

Comment:
We need to be careful here. We compute the calibration factors separately for each measurement from the actual data. The Si fit did not check the parameters for LBCO.